portal-web/test/functional/com/liferay/portalweb/macros/mymacros/MyMacro.macro

```diff
@@ -0,0 +1,37 @@
+definition {
+
+	macro assertScreenCustomValues {
+		AssertTextEquals(
+			locator1 = "MyPath#CUSTOM_SIZE_HEIGHT",
+			value1 = 1920);
+
+		AssertTextEquals(
+			locator1 = "MyPath#CUSTOM_SIZE_WIDTH",
+			value1 = 1080);
+	}
```

Comment:
Hey, I see that you are directly putting values inside the macro, but you can think that: what if you want to assert a different value? you would have to change the macro again, correct?
So try to use, in place of the value1, a variable for the width and for the height, in that way, you will be able to validate any value you want instead of changing the macro.

Also in a real case scenario, what if other people want to use your macro but with different values, but already defined the values? That's why using variables is important. :wink: 

You can use the variables I suggested to you in the comments below, thanks!

Reply:
Thank you! I'll use more variables in cases like this in the future.